tests/models/cpmant/test_tokenization_cpmant.py

```diff
@@ -0,0 +1,42 @@
+# coding=utf-8
+# Copyright 2022 The OpenBMB Team and The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from transformers.testing_utils import is_torch_available
+
+from ...test_tokenization_common import TokenizerTesterMixin
+
+
+if is_torch_available():
+    from transformers.models.cpmant import CpmAntTokenizer
+
+
+@unittest.skip("CPMAntTokenizer process vocab in list format, so we skip the common test.")
```

Comment:
The test below won't be executed because of this global skip here.

Reply:
I skip the test because it's `load_vocab` logic is different from the `TokenizerTesterMixin`. While refactor is not that convenient and necessary i think, so I just skip it. Is it OK? Need I make some changes?

Reply:
No this is not ok, since the test you wrote below will never be executed. If all tests fail in the TokenizerTesterMixin, that means your tokenizer does not have an API consistent with the other tokenizers of Transformers, and thus we can't accept it. You should fix your tokenizer so that it passes most of the tests of the common tester.

Reply:
OK I will fix it, thanks!

Reply:
> The test below won't be executed because of this global skip here.

Solved.